Formula/cayley.rb

```diff
@@ -84,6 +86,20 @@ def plist; <<~EOS
   end
 
   test do
+    # Check version
     assert_match version.to_s, shell_output("#{bin}/cayley version")
+
+    # Check UI
+    begin
+      pid = fork do
+        exec "#{bin}/cayley", "http"
+      end
+      sleep 3
+      response = shell_output("curl -s -i 127.0.0.1:64210")
+      assert_match %r{/HTTP\/1.1 200 OK/}, response
```

Comment:
This can just be a normal string.

Reply:
Maybe I miss something, but this is only one line in the output.

Reply:
Yeah, but assert_match will also work if you feed it a string to match. And in this case that's possible and easier to read.